internal/command/arguments/extended.go

```diff
@@ -78,6 +78,18 @@ type Operation struct {
 	// learn a use-case for broader matching.
 	ForceReplace []addrs.AbsResourceInstance
 
+	// DeferralAllowed enables experimental support for automatically performing
+	// a partial plan if some objects are not yet plannable (due to unknown
+	// values in count/for_each, or due to other missing dependencies that can't
+	// be resolved in a single plan/apply cycle).
+	//
+	// IMPORTANT: This feature should only be available when Terraform is built
+	// with experimental features enabled. Since extendedFlagSet can't currently
+	// test whether experimental features are enabled, the check needs to happen
+	// when _reading_ these Operation arguments and transferring values to the
+	// backendrun.Operation struct.
+	DeferralAllowed bool
```

Comment:
I JUST NOW thought of a potentially nicer way to handle this:

- Use a private field on arguments.Operation.
- `func (o *Operation) DeferralAllowed(experimentsAllowed: bool) bool {...}`

So, require passing in the experiments yea/nay status in order to get at the value, as a safety reminder.

There's only three places that should be touching this anyway, so maybe that's overengineering. But please lmk what you think.